### DISCUSSION TITLE
GSoC 2024: expression of interest

### DISCUSSION THREAD

**Discussion by thibaudcolas** [wagtail/gsoc#111] [Announcements]
👋 we’ve not decided whether we’ll join GSoC in 2024. If you want to participate with us, please register your interest with [our form: wagtail.org/gsoc-apply](https://wagtail.org/gsoc-apply/)

**Comment by ArpanMondalGITHUB** [comment]
Hey fellow programmers, 
                    I am self taught web dev and i know html,css,js,and python and its various framework like django . It would beautifull if i can contribute to this org or i can help you gys . Let me know .

**Comment by pavankalyan767** [comment]
Hey folks, I am a student pursuing computer science and engineering degree I love working on projects that people love to use and currently I am working projects where i use django and other frameworks I would love to be a part of this community by contributing and working on this codebase

**Comment by onkarmane-source** [comment]
Hi,
I am Onkar Mane currently studying in the final year of engineering with Major in Artificial intelligence. I want to contribute to an open source project. I am love join wagtail as my first open source organization

**Comment by Rajasimhareddybolla** [comment]
Hai,
this is Raja from Bangalore and I would like to contribute to open-source projects to boost my Django and approach towards real-world problems. I want to join the Wagtail community as this project is most relevant to my situation. And I would like to participate in GSOC 2024.

**Comment by thibaudcolas** [comment]
👋 for clarity – you should all be using the expression of interest form at this stage. There will be a separate thread for introductions once we’ve actually decided to take part in GSoC.

**Comment by 2001Navneet** [comment]
I am Navneet . I am from India .I am computer science undergrad currently I am in my  third year. I am new to   open  source but I am well aware of html CSS3, JavaScript and React . I would love to contribute to your organization but could you please tell me how to get started?

**Comment by Preetam-Das26** [comment]
Hello I am Preetam from India.I am a Btech undergrad interested in Python,Django,HTML,CSS,JS,C++,JAVA,git,Github,etc. I am interested in this field and have worked on some few projects in these areas involving ML/AI and python baackend .I look forward to your participation in GSoC 2024.If possible please let me know if how to contribute as a beginner to make this journey smooth. Also to get selected as GSoC project contributor which steps should be taken and how to begin? 
Thank you

**Comment by OsafAliSayed** [comment]
Hello Fellow Developers. My name is Osaf Ali Sayed, I am a third-year IIIT Pune student. I am a full-stack web developer using Technologies such as HTML, CSS, JS, React, Angular, TypeScript for frontend and Python, Django, Flask, MySQL, Redis, and PostgreSQL for backend.

I am keenly interested in joining the Wagtail organization for GSoC 2024 and contributing to it as much as possible. I have been trying to understand the codebase for a while now.

Looking forward to working with the core team members and fellow contributers.

**Comment by dainik10** [comment]
Heyy Everyone. I'm Dainik a 3rd yr CSE undergrad. And I'm very much enthusiastic to participate in GSOC'24 as my org as Wagtail. I'm a Full Stack Dev with Django and Python. Happy to connect with you guys.

**Comment by jananivarthini455** [comment]
Heyy everyone ,
          Myself  Janani here And I'm a frontend developer in react.js ,redux, saga ,HTML,CSS,Angular,Typescript. I           would love to contribute in GSoC 2024 with wagtail. hope to work with all core members in future

**Comment by Pranay38** [comment]
Hello Wagtail Community,
I am Pranay Agrawal, a prefinal year student at NIT Raipur, with a strong background in the MERN stack. Alongside my academic pursuits, I have delved into Machine Learning, Deep Learning, and Computer Vision, constantly seeking to explore the artificial intelligence. I am eager to contribute, learn, and engage with like-minded individuals who share an interest in Wagtail and its surrounding technologies.

**Comment by srujan713** [comment]
Heyy Wagtail Community 👋,

I am Srujan Ranpise, a Penultimate year student at PICT Pune, with a strong background in the full stack. 

Beside my academic pursuits, I have delved into data science too with my skills including React.js, SQL, Azure. 

I am eager to contribute, learn, and engage with like-minded individuals who share an interest in Wagtail and its surrounding technologies.

**Comment by shutupatul** [comment]
Hello, I am Atul, an undergraduate student at USICT Delhi. My technical skills are C++, python , DSA, html, css, nodejs, sql and django. I'm looking forward to contribute to wagtail this year in GSOC 2024. Regards.

**Comment by Oliver-Zen** [comment]
Hey there! 👋 This is Oliver Teng. I'm on the path to a CS master's at UChicago. Proficient in Python, SQL, and R, with experience in Java and C. Recently, I crafted a real-time stock trading simulation web app using Flask, Javascript, and Bootstrap. I'm eager and excited to make meaningful contributions to Wagtail, bringing my passion for coding to the community and learning along the way.

**Comment by nikhilkalburgi** [comment]
Hi there!

I'm Nikhil Kalburgi, an experienced open-source contributor. I've made substantial contributions to the Wagtail docs in the past and am eager to extend my involvement to the codebase. Over the last few months, I've actively explored the project structure, gaining a solid understanding. Excited about the prospect of further contributions, I look forward to leveraging my familiarity with the project.

**Comment by aniirathod** [comment]
Hello community 

I'm Aniket Rathod, a undergraduate student. I have the knowledge of Javascript , react and python. I am eager to contribute, learn, and engage with like-minded individuals who share an interest in this community and its surrounding technologies.
I look forward to contributing with all of you.

**Comment by priyanshu16095** [comment]
Hello, I'm Priyanshu Gupta, and interested in contributing to Wagtail for GSOC 2024. I'm in my 1st-year of computer science engineering and familiar with Java, Python, Databases, Design Patterns. I've created some full stack applications using React, Django, Java Spring. I specifically have interest in backend But, now I wanted to improve my skills by doing some open source work and I feel i interested in this project and also my skills align with this.
Thank for reading! I hope i can add some value.

**Comment by akshaydubey05** [comment]
Hello, I am Akshay Kuamr Dubey. I am a backend developer in Python and would like to contribute to Wagtail for GSoC 2024. I am a second-year student at the Indian Institute of Information Technology, Ranchi. I successfully contributed to HacktoberFest 2023 and was able to complete it. I have a good grasp of Django basics and advanced topics and have made many projects using it.I would like to be a part of this community to build great projects for GSoC.

My LiinkedIn-> https://www.linkedin.com/in/akshay-dubey-ba54661b2/

I am hoping for a good response.
Akshay

**Comment by iqraakhtar09** [comment]
Hello mentor, @Stormheg I'm interested in alt text capabilities because,  in one hackathon,, I made text to image generation app and I'm really sure that I'm able to do image to text. But I'm not be able to see Alt text capabilities GitHub kindly provide me a link and let me know I'm going to writing my proposal if there any template available for this please share it with me. When will you available to review my proposal? yeah, I'm full of excitement and potential
here you can find me
https://www.linkedin.com/in/iqra-akhtar-/

**Comment by yuh2k** [comment]
Hey there, I am Hang Yu, currently a master student at Brandeis University major in Computer Science. In the summer of 2023, I interned at TikTok contributed to the new paid content feature - TikTok Series. Over the three months, I took on many responsibilities, including building an API Automation testing framework from scratch using Python and related testing infrastructures. I also developed a range of utilities and tools to streamline our team's workflow and accelerate software delivery. 
I believe my strong skills both in Python and full stack development would be a good fit to the projects @ wagtail, looking foward to making contributions!
Here is my linkedin: https://www.linkedin.com/in/hy22/

**Comment by pooranjoyb** [comment]
Hello @Stormheg , My name is Pooranjoy Bhattachaya, I am writing to express my interest in contributing to Wagtail's Open Source Project for GSOC 2024. I am a 3rd year B.Tech Undergraduate in Computer Science and Engineering at Manipal Institute of Technology.

With a proven track record as a Lead Web Developer at GDSC-SMIT and a portfolio highlighted by innovative projects, I am confident in my ability to contribute effectively to your dynamic team. I am eager to contribute my skills and experiences to the team and be a part of your mission to revolutionize productivity and collaboration. 

Here is my linkedin profile and link to my portfolio. 
https://www.linkedin.com/in/pooranjoy-bhattacharya-baa23721a/

Portfolio : https://pooranjoyb.github.io/portfolio-website/

**Comment by malikrafsan** [comment]
Hi Wagtail Community, my name is Malik Akbar Hashemi Rafsanjani, a final year computer science student from Indonesia (Bandung, Institute of Technology) 👋👋

I am very excited to contribute to this project for Google Summer of Code (GSoC) 2024. I have several experiences, such as internships and freelance projects, regarding web development. Looking forward to working with the community and fellow contributors!

Here is my profile
GitHub: http://github.com/malikrafsan
Personal Website: https://site.malikrafsan.tech/
Linkedin: https://www.linkedin.com/in/malik-rafsanjani/
Email: malikakbarrafsan@gmail.com
Resume: https://drive.google.com/file/d/1Y6hbJb7PhFKJV4yw4BVGk_ZUowybpsZe/view
PPT Profile: https://docs.google.com/presentation/d/1W0UqpdHsrQ8qNBQ_XLc_EVJE9ruEVOK330UHpGhfJ3s/edit

**Comment by vinayJ74** [comment]
hello are you people participating in gsco 2025

**Reply by thibaudcolas** [reply]
Good question! We don’t know yet :) We don’t even know if GSoC as a program will run in 2025. I’ve started a separate thread if you want to subscribe to updates: #123.
